3_keys_instances/c++11/USER_QOS_PROFILES.xml

```
@@ -0,0 +1,80 @@
+    works of the software solely for use with RTI Connext DDS. Licensee may
+    redistribute copies of the software provided that all such copies are
+    subject to this license. The software is provided "as is", with no warranty
+    of any type, including any warranty for fitness for any purpose. RTI is
+    under no obligation to maintain or support the software. RTI shall not be
+    liable for any incidental or consequential damages arising out of the use
+    or inability to use the software.
+
+    This file is used only when it is in the current working directory or when
+    the environment variable NDDS_QOS_PROFILES is defined and points to this
+    file.
+
+    The profile in this file inherits from the builtin QoS profile
+    BuiltinQosLib::Generic.StrictReliable. That profile, along with all of the
+    other built-in QoS profiles can be found in the
+    BuiltinProfiles.documentationONLY.xml file located in the
+    $NDDSHOME/resource/xml/ directory.
+-->
+<dds xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:noNamespaceSchemaLocation="https://community.rti.com/schema/current/rti_dds_qos_profiles.xsd">
+    <!--
+        QoS Library containing the QoS profile used in this example.
+        A QoS library is a named set of QoS profiles.
+    -->
+    <qos_library name="ChocolateFactoryLibrary">
+
+        <!-- 
+            QoS profile used to configure reliable communication between the
+            DataWriter and DataReader created in the example code.
+
+            base_name:
+            Communication is reliable because this profile inherits from
+            the built-in profile "BuiltinQosLib::Generic.StrictReliable"
+
+            is_default_qos:
+            These QoS profiles will be used as the default, as long as this
+            file is in the working directory when running the example.
+        -->
+        <qos_profile name="TemperingTemperatureProfile"
```

The ChocolateLotState readers and writers also use default qos, so the name of this profile and the pub/sub names are misleading.

---

Yeah, I was debating about what to do with this one, since we aren't actually setting QoS explicitly until the next hands on. I'll make it into a more generic name, and then create separate profiles in the next hands-on.